helpers/user_agent_helper.go

```diff
@@ -0,0 +1,72 @@
+package helpers
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+var userAgentPlatform = []func() string{
+	generateFirefoxUserAgent,
+	generateChromeUserAgent,
+}
+
+var firefoxVersionList = []float32{
+	52.0,
+	53.0,
+	54.0,
+	56.0,
+	57.0,
+	58.0,
+	59.0,
+	6.0,
+	60.0,
+	61.0,
+	63.0,
+}
+
+var chromeVersionList = []string{
+	"49.0.2623.112",
+	"55.0.2883.87",
+	"56.0.2924.87",
+	"57.0.2987.133",
+	"61.0.3163.100",
+	"63.0.3239.132",
+	"64.0.3282.0",
+	"65.0.3325.146",
+	"68.0.3440.106",
+	"69.0.3497.100",
+	"70.0.3538.102",
+	"74.0.3729.169",
+	"88.0.4324.182",
+}
+
+var osStrings = []string{
+	"Macintosh; Intel Mac OS X 10_15_5",
+	"Macintosh; Intel Mac OS X 10_10",
+	"Windows NT 10.0",
+	"Windows NT 5.1",
+	"Windows NT 6.1; WOW64",
+	"Windows NT 6.1; Win64; x64",
+	"X11; Linux x86_64",
+}
+
+// RandomUserAgent generates a random DESKTOP browser user-agent on every requests
+func RandomUserAgent() string {
+	return userAgentPlatform[rand.Intn(len(userAgentPlatform))]()
```

Comment:
```
[]func() string{
	generateFirefoxUserAgent,
	generateChromeUserAgent,
}
```

This slice would always have two items, right? `generateFirefoxUserAgent` would generate 1 user agent string and `generateChromeUserAgent` would generate 1 as well.

So technically `userAgentPlatform[2]()` would work, no? 

While the usage of a slice with function seems interesting, it does make it more complex. It can stay as-is but I am concerned if this code is not too smart 😅

Reply:
I've refactored this function to be more clearer from the previous version ✌️